Fixed #33789 -- Doc'd changes in quoting table/column names on Oracle in Django 4.0. (#15784)
django/django: docs/releases/4.0.txt

consequence, any custom deletion logic in ``delete()`` handlers should be
moved to ``form_valid()``, or a shared helper method, if required.

Table and column naming scheme changes on Oracle
------------------------------------------------

Django 4.0 inadvertently changed the table and column naming scheme on Oracle.
This causes errors for models and fields with names longer than 30 characters.

> Unfortunately, it necessarily requires renaming some Oracle tables and columns.

Reviewer: ```suggestion
Unfortunately, renaming some Oracle tables and columns is required.
```

Author: Thanks, updated :+1:
